### DISCUSSION TITLE
Can I change upper/lower case of my username without consequences?

### DISCUSSION THREAD

**Discussion by User670** [community/community#51746] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I kinda want to change my username from `user670` (lowercase u) to `User670` (uppercase U), but I was scared by [this mess](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username) concerning consequences of changing username.

Are usernames case-sensitive? Do I need to worry about those things if I change my username's lower/upper case only?

**Comment by joneslee0918** [comment]
I think you do not need to worry about it.
Because  they are resolving case insensitive way for the user name.
please check it.
https://stackoverflow.com/questions/58591331/how-to-change-the-case-of-my-githubs-username-without-changing-any-letter

**Comment by NJacksonDev** [comment]
The username should be case insensitive (upper/lowercase does not affect anything). See below for relevant stackoverflow discussion.

https://stackoverflow.com/questions/58591331/how-to-change-the-case-of-my-githubs-username-without-changing-any-letter

**Comment by tdsreader** [comment]
Is it possible to make changing the case of the username NOT counted as change of the username, at least in the sense of not being able to change back to previous name for 90 days, but just changing the DISPLAY username?

**Comment by OverlordMIke** [comment]
I want to be able to fix the errant upper case I in my name/url. I made this account years ago and have no idea how I managed to do that and now github insists that the "I" is always uppercase lol

I know it makes no difference functionally. But it just looks bad, y'know?

**Comment by benaiad** [comment]
>It appears to now be possible to change the case of your username without changing to some other name first.

https://github.com/orgs/community/discussions/22822
